admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -79,9 +79,7 @@
              title="{$smarty.capture.title|htmlspecialchars}"
              href="{$link->getAdminLink('AdminMaintenance')|escape:'html':'UTF-8'}"
           >
-            <i class="material-icons"
-              style="{if isset($maintenance_allow_admins) && $maintenance_allow_admins}color: var(--green);{/if}"
```

Comment:
You have to keep it.

Reply:
I have highlighted the maintenance link in the header in CSS after a talk with designers ✅